business_notes.md

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
I would consider putting this inside a new directory maybe `/notes`? I think it's generally good practice to try to keep the root folder of the repository as clean as possible.

Reply:
done